csrc/executor.cpp

```diff
@@ -524,7 +524,7 @@ std::pair<std::vector<int64_t>, std::vector<int64_t>> inferShape(
 
   std::vector<int64_t> concrete_sizes(symbolic_sizes.size(), 0);
 
-  for (const auto i : c10::irange(symbolic_sizes.size())) {
+  for (const auto i : std::views::iota((size_t)0, symbolic_sizes.size())) {
```

Comment:
Similarly, consider static_cast.

Reply:
Generally, I agree to prefer `static_cast` over C style casts. But for this specific case, I prefer `(size_t)0` over `static_cast<size_t>(0)`. The only reason is because `static_cast<size_t>(0)` is too verbose, and I want to keep writing a simple loop simple. In my opinion, `(size_t)0` is trivial enough, and can hardly be wrong. If we have to spell out all the `static_cast<size_t>(0)`, I don't see any reason why we do not just do `for (size_t i = 0; i < blabla; i++)`. Similar to https://github.com/NVIDIA/Fuser/pull/1017#discussion_r1345165901, it is unfortunate that C++ prefers vague names like `size_t`, but does not provide a way to create a literal of type `size_t`.